aws/data_source_aws_ecr_repository.go

```diff
@@ -25,12 +25,28 @@ func dataSourceAwsEcrRepository() *schema.Resource {
 			},
 			"registry_id": {
 				Type:     schema.TypeString,
-				Computed: true,
+				Optional: true,
```

Comment:
This argument needs to keep `Computed` if `d.Set()` is being called on it, see also: https://github.com/terraform-providers/terraform-provider-aws/pull/14412

```suggestion
				Optional: true,
				Computed: true,
```

These types of errors will start showing when we switch the Terraform binary testing framework in SDK v2 and configure it for 0.13.